maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -251,20 +251,20 @@ class IOSDriver(
         when (swipeDirection) {
             SwipeDirection.UP -> {
                 iosDevice.scroll(
-                    xStart = width / 4,
-                    yStart = height,
-                    xEnd = width / 4,
-                    yEnd = height / 4,
```

Comment:
Looks like there are some inconsistencies with the Android implementation. Let's keep them consistent unless there are reasons to have different behaviors between platforms